projects/apple/fixup_bundle.py

```diff
@@ -122,6 +122,12 @@ def _getdependencies(path):
   return val.split()
 
 def isexcluded(id):
+  # we don't consider the gcc libraries as system libraries on MacOS because
+  # Apple no longer distributes gcc.  If these are there they were installed by
+  # the user.  They will probably be there as we typically use gfortran to compile
+  # scipy and we need to pull them into the bundle for scipy to be redistributable.
+  if re.match(r".*/gcc/.*", id):
```

Comment:
The `/Cellar/` is actually what we don't want.  brew tries to fix up the installed libraries so they don't link to `/Cellar/` and they missed one.  There were links to `libquadmath.0.dylib` in two locations, on in `/Cellar/` and one in `/usr/local/lib/...`  That is what caused the second problem we found, fixup bundle thought they were two separate libraries and left a dangling link to the one in `/Cellar/`.